gcc/rust/checks/errors/rust-feature-gate.h

```diff
@@ -21,6 +21,8 @@
 
 #include "rust-ast-visitor.h"
 #include "rust-ast-full.h"
+#include "rust-feature.h"
+#include <unordered_set>
```

Comment:
You should avoid adding includes for stl like this. Please rely on rust-system.h std::set should already be available.